Ska/Shell/tests/test_shell.py

```diff
@@ -65,22 +65,22 @@ def test_bash_shell(self):
         assert env == {}
 
     def test_env(self):
-        envs = getenv('export TEST_ENV_VAR="hello"')
-        assert envs['TEST_ENV_VAR'] == 'hello'
-        outlines = bash('echo $TEST_ENV_VAR', env=envs)
+        envs = getenv('export TEST_ENV_VARA="hello"')
+        assert envs['TEST_ENV_VARA'] == 'hello'
+        outlines = bash('echo $TEST_ENV_VARA', env=envs)
         assert outlines == ['hello']
 
     def test_importenv(self):
-        importenv('export TEST_ENV_VAR="hello"', env={'TEST_ENV_VAR2': 'world'})
-        assert os.environ['TEST_ENV_VAR'] == 'hello'
-        assert os.environ['TEST_ENV_VAR2'] == 'world'
+        importenv('export TEST_ENV_VARC="hello"', env={'TEST_ENV_VARB': 'world'})
+        assert os.environ['TEST_ENV_VARC'] == 'hello'
+        assert os.environ['TEST_ENV_VARB'] == 'world'
 
     def test_logfile(self):
         logfile = StringIO()
-        bash('echo line1; echo line2', logfile=logfile)
-        logfile.seek(0)
-        outlines = logfile.read().splitlines()
-        assert outlines[0].startswith('Bash-')
+        cmd = 'echo line1; echo line2'
+        bash(cmd, logfile=logfile)
+        outlines = logfile.getvalue().splitlines()
+        assert outlines[0].endswith(cmd)
```

Comment:
There seems to be a difference between Py2 and Py3 with regards to whether the prompt gets logged here. But what we care about is the command.